scrapers/Clips4Sale.yml

```diff
@@ -10,7 +10,30 @@ sceneByURL:
       - clips4sale.com/search/clip/
     scraper: c4sClipsSceneScraper
 
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://www.clips4sale.com/clips/search/{}
+  scraper: c4sSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: c4sSceneScraper
+
 xPathScrapers:
+  c4sSearch:
+    common:
+      $card: //figure[contains(@class, "splide__slide")]
+    scene:
+      Title:
+        selector: $card/@aria-label
+      URL:
+        selector: $card//a[contains(@class, "search-clip__title")]
```

Comment:
Found the error. This xpath resolves to the inner text, not the URL. You want to append `/@href` on the end.

Reply:
Thanks, pushed a fix :)